ProcessLib/ThermoHydroMechanics/CreateThermoHydroMechanicsProcess.cpp

```diff
@@ -0,0 +1,310 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateThermoHydroMechanicsProcess.h"
+
+#include <cassert>
+
+#include "MaterialLib/SolidModels/MechanicsBase.h"
+#include "MaterialLib/SolidModels/CreateConstitutiveRelation.h"
+#include "ProcessLib/Output/CreateSecondaryVariables.h"
+#include "ProcessLib/Utils/ProcessUtils.h"
+
+#include "ThermoHydroMechanicsProcess.h"
+#include "ThermoHydroMechanicsProcessData.h"
+
+namespace ProcessLib
+{
+namespace ThermoHydroMechanics
+{
+template <int DisplacementDim>
+std::unique_ptr<Process> createThermoHydroMechanicsProcess(
+    MeshLib::Mesh& mesh,
```

Comment:
Please, check if the reference can be `const`, i.e.: `MeshLib::Mesh const& mesh`.

Reply:
I have checked... There will be compiling errors if const is used...